builtin/logical/pki/path_roles.go

```diff
@@ -504,6 +529,7 @@ type roleEntry struct {
 	OU                    string `json:"ou" structs:"ou" mapstructure:"ou"`
 	Organization          string `json:"organization" structs:"organization" mapstructure:"organization"`
 	GenerateLease         *bool  `json:"generate_lease,omitempty" structs:"generate_lease,omitempty"`
+	NoStore               *bool  `json:"no_store,omitempty" structs:"no_store,omitempty"`
```

Comment:
This doesn't need to be a `*bool`, just `bool` is fine, with associated updates elsewhere.